ui/components/app/confirm-page-container/confirm-page-container-content/confirm-page-container-content.component.js

```diff
@@ -15,6 +15,7 @@ export default class ConfirmPageContainerContent extends Component {
   static propTypes = {
```

Comment:
The changes in this file allow for the addition of the "Hex" tab alongside the "Data" tab as shown here:
![Screenshot from 2021-12-01 09-39-08](https://user-images.githubusercontent.com/7499938/144240129-f3c17507-93a5-4926-8936-adf2f028a250.png)